rest-api-spec/src/main/resources/rest-api-spec/api/ml.stop_trained_model_deployment.json

```diff
@@ -26,6 +26,21 @@
           }
         }
       ]
+    },
+    "params":{
+      "allow_no_match":{
+        "type":"boolean",
+        "required":false,
+        "description":"Whether to ignore if a wildcard expression matches no deployments. (This includes `_all` string or when no deployments have been specified)"
+      },
+      "force":{
+        "type":"boolean",
+        "required":false,
+        "description":"True if the deployment should be forcefully stopped"
+      }
+    },
+    "body":{
+      "description":"The stop deployment parameters"
```

Comment:
What sorts of options can be passed in the `body`? I don't see a request body documented for this API.

Reply:
There isn't one. I am not a HUGE fan of this convention (its elsewhere in ES), where the query params can optionally be supplied in the body. I would much rather have one or the other, but this change does follow convention.